QUEST.tsv

```diff
@@ -92,10 +92,10 @@ xQUEST_20150317_000091	And yet, it doesn't matter. Come out and face me, you mon
 xQUEST_20150317_000092	Just now, I saw a goddess appear in my dream… {nl}How surreal…
 xQUEST_20150317_000093	Still. I have a duty to defend the kingdom. It's why I'm here now.
 xQUEST_20150317_000094	Uska's an old friend of mine.
-xQUEST_20150317_000095	We did our tests together, and Uska got to where he is now with my support. {nl}Oh, those were some crazy days… It'd be nice to get together again!
-xQUEST_20150317_000096	Just don't believe some of the things he'll say. {nl}His hometown knew him for being quite the show-off.
+xQUEST_20150317_000095	We did our tests together, and Uska only got to where he is now with my help. {nl}Oh, those were some crazy days… It'd be nice to get together again!
+xQUEST_20150317_000096	Just don't believe some of the things he says. {nl}He had a reputation in his hometown for being quite the show-off.
 xQUEST_20150317_000097	Me? I find it funny, so I just leave him alone. 
-xQUEST_20150317_000098	Uh, wait. I'll take back what I said about not believing a few things.
+xQUEST_20150317_000098	Uh, wait {nl}I take back what I said about not believing a only a few things.
```

Comment:
The original Korean line has the {nl} tag at the end of the line after the period. Try not to randomly place {nl} tags because we don't know how it will look in-game. https://github.com/Treeofsavior/EnglishTranslation/issues/526